main/tests/server/src/com/google/refine/operations/column/ColumnAdditionByFetchingURLsOperationTests.java

```diff
@@ -162,14 +154,11 @@ public void testUrlCaching() throws Exception {
                     OnError.StoreError,
                     "rand",
                     1,
-                    500,
+                    5,
                     true,
                     null);
 
-            // We have 100 rows and 500 ms per row but only two distinct
-            // values so we should not wait much more than ~1000 ms to get the
-            // results.
-            runAndWait(op, 1500);
+            runOperation(op, project);
```

Comment:
Is this testing the functionality implied by the previous comment about collapsing identical values?